_data/application.yml

```diff
@@ -302,53 +302,73 @@
 
   - group: proposed-occupancy
     fields:
-    - label: Proposed occupancy codes
+    - label: What will the building’s occupancy codes be after construction?
       type: checkbox
       options:
-      - label: "R-1: Residential, transient hotel / motel (residential)"
-      - label: "R-2: Residential, apartment / condominium with 3+ units"
-      - label: "R-2.1: Residential, supervised apartment / condominum"
-      - label: "R-3: General residential"
-      - label: "R-3.1: General supervised residential"
-      - label: "R-4: Residential, supervised"
-      - label: "S: Storage"
+      - label: "**R-1**: Residential, transient hotel / motel (residential)"
+      - label: "**R-2**: Residential, apartment / condominium with 3+ units"
+      - label: "**R-2.1**: Residential, supervised apartment / condominum"
+      - label: "**R-3**: General residential"
+      - label: "**R-3.1**: General supervised residential"
+      - label: "**R-4**: Residential, supervised"
+      - label: "**S**: Storage"
       - label: "Other"
         shows: proposed occupancy-other
       annotation: DBI and Planning
 
     - group: occupancy-proposed-other
       fields:
-        - label: Check off the rest of the building's proposed occupancy codes.
+        - label: Check all of the building's occupancy codes after construction.
           type: checkbox
           options:
-          - label: "A-1: Assembly, fixed seating"
```

Comment:
The text for all of these options are the same, but the code numbers are now in bold.